### ISSUE TITLE
Write some documentation

### ISSUE THREAD

**Issue by killian-mahe** [ElliotThiebaut/ettt-api#15]
It could be really useful to write some documentation (using [Swagger](https://swagger.io/) for example) to help developers understand your API and its responses.

**Comment by ElliotThiebaut** [comment]
The docs are available at [](https://ettt-cloud.stoplight.io/docs/ettt-cloud-api), but a README.md should be added to give the information.

**Comment by killian-mahe** [comment]
Okay, so you have it. All we have to do is add it to a nice and beautiful README.md !

**Comment by ElliotThiebaut** [comment]
The readme should include standard stuff about NodeJS but also a "How to contribute" section or something like that.

**Comment by killian-mahe** [comment]
Yes, and a section or a sub-section "How to install and launch".
